backend/lib/models/Post.js

```diff
@@ -4,6 +4,8 @@ const { Schema, model, ObjectId } = require("mongoose");
 const EXPIRATION_OPTIONS = ["day", "week", "month", "forever"];
 const VISIBILITY_OPTIONS = ["city", "country", "state", "worldwide"];
 const POST_OBJECTIVES = ["request", "offer"];
+const SORT_OPTIONS = ["likes", "updatedAt", "views", "shares"];
```

Comment:
just realized these 2 options are for the searching feature only, it seems they're not attributes of a `Post`, so maybe it's better to define them in the `lib/endpoints/schema/posts.js` instead ?

Reply:
I will resolve this in the next branch where we fix the sorting exclusivity issue on the frontend